variables.tf

```diff
@@ -1,10 +1,11 @@
+
 /* general */
-variable "hosts" {
+variable "node_count" {
   default = 3
 }
 
 variable "domain" {
-  default = "example.com"
+  default = "stokebrain.com"
```

Comment:
Please revert

Reply:
This was an accident, which I thought I had caught and fixed.